docs/user-guides/observability-ui-plugins.md

```diff
@@ -20,3 +22,33 @@ metadata:
 spec:
   type: Dashboards
 ```
+
+### Troubleshooting Panel
+
+The plugin will connect to a Korrel8r instance named `korrel8r` in the `korrel8r` namespace. A "Troubleshooting Panel" button is added to the alerts page, which will convert the current alert into a Korrel8r query, then retrieve related neighbors and display them in a topology view.
```

Comment:
This is not accurate but can be changed with #497 by @shwetaap

Reply:
Changed it in the PR https://github.com/rhobs/observability-operator/pull/497